src/app/games/pentago/pentago.component.ts

```diff
@@ -63,25 +63,21 @@ export class PentagoComponent extends RectangularGameComponent<PentagoRules,
     public updateBoard(): void {
         this.board = this.getState().getCopiedBoard();
         this.victoryCoords = this.rules.getVictoryCoords(this.getState());
-        this.showLastMove();
+        if (this.rules.node.move.isAbsent()) {
+            this.hidePreviousMove();
+        }
```

Comment:
always hide & rely on showLastMove to be called when needed?

Reply:
removed altogether !